crown/src/nockapp/mod.rs

```diff
@@ -423,24 +429,24 @@ impl NockApp {
             res = tasks_fut => {
                 match res {
                     Some(Ok(Err(e))) => {
-                        eprintln!("{e:?}")
+                        error!("{e:?}")
                     },
                     Some(Err(e)) => {
-                        eprintln!("{e:?}")
+                        error!("{e:?}")
                     },
                     _ => {},
                 }
             },
             action_res = self.action_channel.recv() => {
                 if let Some(action) = action_res {
-                    println!("action: {:?}", action);
+                    info!("action: {:?}", action);
                     match action {
                         IOAction::Poke { poke, ack_channel } => {
-                            println!("poke slab: {:?}", poke);
+                            info!("poke slab: {:?}", poke);
                             let poke_noun = poke.copy_to_stack(self.kernel.serf.stack());
-                            println!("poke_noun: {:?}", poke_noun);
+                            info!("poke_noun: {:?}", poke_noun);
                             let effects_res = self.kernel.poke(poke_noun);
-                            println!("effects_res: {:?}", effects_res);
+                            info!("effects_res: {:?}", effects_res);
```

Comment:
Not sure how info! works, but this would be an issue in some settings with printing large nouns. Good for debugging, but also important to be able to turn off the print. Does the "release" version turn these off?

Reply:
i think we default to setting the log level to info, so i'll change this to `debug!`. that will turn it off for the release version, and it can be enabled by setting `--log-level` or using `RUST_LOG`

Reply:
https://docs.rs/tracing/latest/tracing/struct.Level.html#impl-Level

My guess is `trace!` is most appropriate for this, rather than `info!`